exercises/webapp/src/components/HackerNews.spec.js

```diff
@@ -0,0 +1,70 @@
+import {
+  mount
+} from '@vue/test-utils';
+import HackerNews from './HackerNews.vue';
+import NewsItem from './NewsItem.vue';
+
+describe("Sorting Order", () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = mount(HackerNews, {
```

Comment:
You could consider to use a setup method:
```js
const createWrapper = (options) => {
  const defaults = { /* ... */ }
  return mount(HackerNews, { ...defaults, ...options })
}
```
Which gives extra flexibility in your test cases:
```js
wrapper = createWrapper({just: 'this change'})
```